app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/models/Order.java

```diff
@@ -1,55 +1,92 @@
 package com.epfl.neighborfood.neighborfoodandroid.models;
 
+import java.text.ParseException;
 import java.util.Date;
-
+import java.text.SimpleDateFormat;
 public class Order extends Model {
 
     private static final String ORDER_DELIVERED = "This order was achieved the";
     private static final String ORDER_NOT_YET_DELIVERED = "This order is being processed";
-    private Meal meal;
-    private Date orderDate;
+    private String orderId;
+    private String mealId;
+    private String orderDate;
     private boolean status;

```

Comment:
the name of the field should be more explicit, what do you mean by status, processed, in process?